### ISSUE TITLE
readme.md?

### ISSUE THREAD

**Issue by Quoteme** [matlorr/bloggy#1]
Das projekt sieht interessant aus, schon alleine, weil es in haskell geschrieben ist.
Was macht es? Könntest du eine Readme.md hinzufügen?

**Comment by matlorr** [comment]
Klar, werde ich machen. Habe vergessen, dass dieses Projekt public ist. Der hier gezeigt Code wurde als IHP-Projekt gebaut
das Tutorial ist sehr zu empfehlen! https://ihp.digitallyinduced.com/Guide/your-first-project.html
